egs/tedlium/s5_r2/local/chain/run_tdnn.sh

```diff
@@ -149,7 +149,7 @@ if [ $stage -le 18 ]; then
     --chain.leaky-hmm-coefficient 0.1 \
     --chain.l2-regularize 0.00005 \
     --chain.apply-deriv-weights false \
-    --chain.lm-opts="--num-extra-lm-states=2000" \
+    --chain.lm-opts="--ngram-order=4 --num-extra-lm-states=2000" \
```

Comment:
can you please remove the --ngram-order=4  since it's the default?  Then should be ready to merge.